app/main.development.js

```diff
@@ -673,7 +673,7 @@ app.on("ready", async () => {
   if (!locale) {
     let newCfgLocale = appLocaleFromElectronLocale(app.getLocale());
```

Comment:
You might wanna copy/adjust something like this on `LanguageSelectPage.getInitialState()`, so that the language select shows by default the user's OS language, instead of being `selector.sortedLanguages[0]`.

Another option would be to add a selector such as `export const defaultLocaleName = () => appLocaleFromElectronLocale(...)`, and pass that along with the `walletSetup` connector and use that when `currentLocaleName` is empty (to make even the initial start page appear on the user's language by default).